packages/camera/lib/src/utils.dart

```diff
@@ -0,0 +1,51 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:camera_platform_interface/camera_platform_interface.dart';
+import 'package:flutter/services.dart';
+
+/// Parses a string into a corresponding CameraLensDirection.
+CameraLensDirection parseCameraLensDirection(String string) {
+  switch (string) {
+    case 'front':
+      return CameraLensDirection.front;
+    case 'back':
+      return CameraLensDirection.back;
+    case 'external':
+      return CameraLensDirection.external;
+  }
+  throw ArgumentError('Unknown CameraLensDirection value');
```

Comment:
This file is just a copy of https://github.com/flutter/plugins/blob/main/packages/camera/camera_android/lib/src/utils.dart and https://github.com/flutter/plugins/blob/main/packages/camera/camera_platform_interface/lib/src/utils/utils.dart. Also I don't see any problem with the code. (Maybe you're talking about the style difference?)

Reply:
I don't think the code has any problem.

However, I was inquired about whether there was a different intention because it was different from the other functions dealing with `throw` in the format as shown below.

<pre>
 default:
     throw ArgumentError('something');
</pre>

Thank you for answer.

Reply:
The original code has been written by different authors. I think that's the reason for the difference. For me they just look logically identical.